reVX/least_cost_xmission/costs/landfall_cost_creator.py

```diff
@@ -0,0 +1,46 @@
+"""
+Create wet (offshore) costs and save to GeoTIFF.
+"""
+import logging
+
+import numpy as np
+
+from reVX.least_cost_xmission.layers.base import LayerCreator
+from reVX.least_cost_xmission.config.constants import (
+    LANDFALL_COSTS_TIFF, LANDFALL_COSTS_H5_LAYER_NAME
+)
+
+logger = logging.getLogger(__name__)
+
+
+class LandfallCostCreator(LayerCreator):
+    """
+    Create landfall costs and save to GeoTIFF.
+    """
+
+    def build(self, landfall_cost: float,
+              landfall_layer_name: str = LANDFALL_COSTS_H5_LAYER_NAME):
+        """
+        Build complete offshore costs. This is currently very simple. In the
```

Comment:
Docstring appears copy-pasted from offshore costs and is somewhat erroneous

Reply:
Oops, good catch. Will update